src/commandHandlers/stats.ts

```diff
@@ -0,0 +1,20 @@
+import { Client, MessageEmbed } from 'discord.js';
+
+import { StatsService } from './stats.service';
+
+export const command = async (client: Client, embed: MessageEmbed) => {
+    const statsService = new StatsService(client)
+
+    const memberCount = await statsService.getServerMemberCount();
+    const totalMessages = await statsService.getServerTotalMessages();
+    embed
+        .setTitle('Server stats')
+        .addField('total users', memberCount)
+        .addField('total messages', totalMessages)
+
+    return embed;
+};
+
+export const description = 'Server status';
```

Comment:
@garretcharp is this description still used?

Reply:
Yes the description is used for the help command

Reply:
Maybe we could document this so that a new developer knows how to create a new command, with the same rules as other commands, to keep consistency. Maybe on the README file, a section saying **Creating a new command hander**. We can have a rule that says "use camelCase for file names in /commandHandlers" or maybe "use dashes for file names".

Do you guys think that type of documentation is valuable @eddiejaoude and @garretcharp ?

Reply:
Yep great idea 👍

Reply:
@BOLT04 I would say camelCase is the best and something we already do with `codeOfConduct.ts`. I think adding it in would be of use most likely.